test/dummy/config/database.yml

```diff
@@ -15,8 +15,6 @@ development:
 test:
   adapter: postgresql
   database: lit_test
-  username: lit
```

Comment:
If we keep this checked in, could we please have it read from an environment variable (with the file then living as `.yml.erb`) or something? Or let's just rename it `database.yml.sample` and add `database.yml` to `.gitignore`.

Reply:
Updated and moved to database.yml.sample w/o the user/pass as db:setup won't create pg users for you.